openedx/core/lib/request_utils.py

```
@@ -120,34 +120,56 @@ def __call__(self, request):
 #     ``rest_framework.exceptions.PermissionDenied``.
 
-# .. toggle_name: EXPECTED_ERRORS[N]['IS_IGNORED']
+# .. setting_name: ignored_ERRORS[N]['MODULE_AND_CLASS']
+# .. setting_default: None
+# .. setting_description: Required error module and class name that is ignored. For example,
+#     ``rest_framework.exceptions.PermissionDenied``.
+
+# .. toggle_name: ignored_ERRORS[N]['IS_IGNORED']
+# .. toggle_implementation: DjangoSetting
+# .. toggle_default: True
+# .. toggle_description: Set this to False if the errors are not ignored by monitoring, but only ignored, like
+#      for temporary problems that may take some time to fix. If True, adds the custom attributes
+#      ``error_ignored_class`` and ``error_ignored_message`` to help diagnose issues with ignored errors, since
+#      this data is not otherwise available. For example of ignoring errors in New Relic, see:
+#      https://docs.newrelic.com/docs/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected/#ignore  pylint: disable=line-too-long,useless-suppression
+#      To query for ignored errors, you would use ``error_ignored_class IS NOT NULL``.
+#      Note: This is defaulted to True because it will be easier for us to detect if True is not the correct value, by
+#      seeing that these errors aren't actually ignored.
+# .. toggle_warning: At this time, this toggle does not actually configure the error to be ignored. It is meant to match
+#     the ignored error configuration found elsewhere. When monitoring, no errors should ever have the attribute
+#     ``error_ignored_class``. Only Transactions should have this custom attribute. If found for an error, it means we
+#     are stating an error should be ignored when it is not actually configured as such, or the configuration is not
+#     working.
+# .. toggle_use_cases: opt_out
+# .. toggle_creation_date: 2021-03-11
+
+# .. toggle_name: IGNORED_ERRORS[N]['IS_IGNORED']
```

This sub-toggle (`'IS_IGNORED'`) is going away and should be deleted entirely. That includes any references to `IS_IGNORED` in the code. It will always be True.

Note: It used to be True for ignored errors, or set to False for expected errors. However, since we are removing expected errors, this isn't needed anymore.

---

This still has not been completed. Let me know if it isn’t clear. This whole toggle and it’s annotations need to be deleted.